pdf_parser/objects/PdfObjects.py

```diff
@@ -185,16 +185,16 @@ def get_lines_by_keywords(self, keywords, context=0):
             lines.extend(page.lines)
         ac_automaton.make_automaton()
         for num, line in enumerate(lines):
-            for index, value in ac_automaton.iter(line.text):
+            for index, value in ac_automaton.iter(line.text.lower()):
```

Comment:
was that the sneaky line that was producing the error?

Reply:
Yep, here and `re.IGNORECASE` just under it too